app/components/termsoverlay/termsoverlay.js

```diff
@@ -13,82 +13,217 @@
  * You should have received a copy of the License along with this program; if
  * not, you can obtain one from Tidepool Project at tidepool.org.
  */
-
 var React = require('react');
+var LoginNav = require('../loginnav');
 
 var TermsOverlay = React.createClass({
   propTypes: {
     onSubmit: React.PropTypes.func,
-    trackMetric: React.PropTypes.func.isRequired
+    trackMetric: React.PropTypes.func.isRequired,
+    ages: React.PropTypes.object,
```

Comment:
If you're defaulting these props, might as well make them `isRequired` - will never fail (and I prefer/I think it's best practice to require props whenever possible). So related, can we also require `onSubmit` or no?